media/sdb/sysmain.xml

```diff
@@ -283,38 +283,47 @@
         </LAYER>
         <LAYER NAME="VISTARTM">
             <SHIM_REF NAME="VistaRTMVersionLie" />
+            <DATA NAME="SHIMVERSIONNT" DATA_DWORD="600" />
```

Comment:
Base 10 to stay compatible with the data windows stores in this tag.

Reply:
This means eg. that for version "7" they have "700" and for version "10" they have "1000"? good :)